span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/fieldgenerators/FieldsGenerator.java

```diff
@@ -42,9 +43,13 @@ private void initializeProtocolFieldGeneratorsMap() {
         .forEach(k -> protocolFieldsGeneratorMap.put(k, rpcFieldsGenerator));
   }
 
-  public void populateOtherFields(Event.Builder eventBuilder) {
+  public void populateOtherFields(
```

Comment:
Pl. add comments that this method assumes that attributeValueMap containers all raw spans tags.